src/ripple/protocol/STObject.h

```diff
@@ -440,7 +437,6 @@ class STObject
     STPathSet const& getFieldPathSet (SField const& field) const;
```

Comment:
Just an observation that `getFieldPathSet` remains while `setFieldPathSet` is removed.